script.js

```diff
@@ -0,0 +1,47 @@
+let btn = document.getElementById('btn');
+let dialog = document.getElementById('showDialog');
+btn.style.position = "absolute";
+
+function crazyButton() {
+    btn.style.left= (Math.random()*90)+"%";
+    btn.style.top= (Math.random()*90)+"%";
+}
+function closeDialog() {
+    dialog.close();
+}
+function showConfetti(){
+    dialog.showModal();
+    document.getElementsByClassName('confetti-wrapper')[0].style.display="block";
+    document.getElementsByClassName('celebrate')[0].style.display="block";
```

Comment:
I changed 
    ` document.getElementById('confetti-wrapper').style.display = "block";`
to this
    `document.getElementsByClassName('celebrate')[0].style.display="block";`